basil/HL/binder_mk53.py

```diff
@@ -45,21 +37,36 @@ def __init__(self, intf, conf):
 
     def init(self):
         super(binderMK53, self).init()
+        # Operation addresses
+        self.ADDR_CURTEMP = 0x11a9
+        self.ADDR_DOOROPEN = 0x1007  # does not work, wrong address?
+        self.ADDR_SETPOINT = 0x1077
+        self.ADDR_MANSETPT = 0x1581
+        self.ADDR_BASICSETPT = 0x156f
+        self.ADDR_MODE = 0x1a22
```

Comment:
Why did you move the `ADDR` attributes into the instance but not the `FUNCTION` ones and `ERROR_CODES`? If you want to be consistent I also would put these attributes in the class (since they are the same for every instance) like so

    class BinderMK53(HardwareLayer):

        # MODBus function codes
        FUNCTION_READN = 0x03  # read n words
        FUNCTION_READN_ALT = 0x04  # read n words, does not occur
        FUNCTION_WRITE = 0x06  # write word
        FUNCTION_WRITEN = 0x10  # write n words

        # Operation addresses
        ADDR_CURTEMP = 0x11a9
        ADDR_DOOROPEN = 0x1007  # does not work, wrong address?
        ADDR_SETPOINT = 0x1077
        ADDR_MANSETPT = 0x1581
        ADDR_BASICSETPT = 0x156f
        ADDR_MODE = 0x1a22

        # Error codes
        ERROR_CODES = {
            1: "Invalid function",
            2: "Invalid parameter address",
            3: "Pparameter value outside range of values",
            4: "Slave not ready",
            5: "Write access to parameter denied"
        }

        def init(self):
            super(binderMK53, self).init()

Now all off these are avbailable in each instance via `self.ADDR*`, `self.FUNCTION*` and `self.ERROR_CODES`.
The advantage is now that you do not have to define everything again for your BinderMK56 (which is a subclass of BinderMK53) since almost everything, except 3 `ADDR` attributes, is the same

Reply:
Done.